### ISSUE TITLE
[BUG] On value updated old signal bindings are not released from widgets causing previous handlers execution on trigger

### ISSUE THREAD

**Issue by stremlenye** [elkowar/eww#443]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

In the configuration like 
```lisp
(button 
      :onclick "echo ${value}"
      "first button")
    (button 
      :onclick "echo ${second-value}"
      "second button")
```

updating the `value` variable causes the signal bindings not to be released causing the previously
bound handlers to be executed along with the most recent ones.

### Reproducing the issue

_eww.yuck_
```lisp
(defvar value "I - 1")
(defvar second-value "II - 1")


(defwindow test
  :monitor 0
  :geometry (geometry 
    :x "34%"
    :y "-40px"
    :width "250px"
    :height "230px"
    :anchor "bottom center")
  (box
    :orientation "v"
    :spacing 10
    (button 
      :onclick "echo ${value}"
      "first button")
    (button 
      :onclick "echo ${second-value}"
      "second button")
))
```

_test.sh_
```sh
eww -c ~/.config/eww kill && \
eww -c ~/.config/eww daemon && \
eww -c ~/.config/eww open test && \ 
eww -c ~/.config/eww update value="I - 2" && \
eww -c ~/.config/eww update second-value="II - 2" && \
eww -c ~/.config/eww update second-value="II - 3" && \
echo "done"
```

Then click on both buttons.

The logs would show:

```sh
(eww:1904136): GLib-GObject-WARNING **: 19:29:40.611: ../../../gobject/gsignal.c:2736: instance '0x55e02a33a340' has no handler with id '37'

(eww:1904136): GLib-GObject-WARNING **: 19:29:40.636: ../../../gobject/gsignal.c:2736: instance '0x55e02a33a180' has no handler with id '45'

(eww:1904136): GLib-GObject-WARNING **: 19:29:40.648: ../../../gobject/gsignal.c:2736: instance '0x55e02a33a340' has no handler with id '80'

I - 1
I - 2
II - 1
II - 3
```

Such behaviour does not manifest itself until the `value` variable is updated.
Basically, it is triggered by updating the value bound to the top-most button.

### Expected behaviour

The previously bound handlers are released on variable updates.

```sh
I - 2
II - 3
```

**Comment by elkowar** [comment]
should be fixed with that merge now!
